src/feature-detection.js

```diff
@@ -16,13 +16,11 @@ const browserEngine = parser.getEngineName()
 const webKit = browserEngine === 'WebKit'
 
 /**
- * Check selectionchange event (currently supported in IE, Chrome and Safari)
- *
- * To handle selectionchange in firefox see CKEditor selection object
- * https://github.com/ckeditor/ckeditor-dev/blob/master/core/selection.js#L388
+ * Check selectionchange event (currently supported in IE, Chrome, Firefox and Safari)
+ * Firefox supports it since version 52 (2017) so pretty sure this is fine.
  */
 // not exactly feature detection... is it?
-export const selectionchange = !(browserEngine === 'Gecko' || browserName === 'Opera')
+export const selectionchange = !(browserName === 'Opera')
```

Comment:
We could replace this with a feature detection:
https://github.com/2is10/selectionchange-polyfill/blob/master/selectionchange.js#L42-L54

```
function hasNativeSupport (doc = window.document) {
  var osc = doc.onselectionchange;
  if (osc !== undefined) {
    try {
      doc.onselectionchange = 0
      return doc.onselectionchange === null
    } catch (e) {
    } finally {
      doc.onselectionchange = osc
    }
  }
  return false
}
```

Reply:
I would vote for not supporting Opera anymore. What do you think @peyerluk ?

Reply:
i thought newer opera versions  are based on webkit. 
from wikipedia;
> In 2013, Opera switched from the Presto engine to Chromium.

Reply:
I don't see any reason to support browsers that can't keep up with the spec. I'dimplement the feature detection here but drop support for legacy browsers soon. 

@TorstenDittmann what browser versions do you need to support?

Reply:
Nothing too specific, technically I need to support Chrome and Safari to support Electron/Capacitor.

The rest I don't really mind to be fair +:1: